### ISSUE TITLE
project.json deprecated

### ISSUE THREAD

**Issue by ericnewton76** [richorama/nuget-tree#9]
Planning to update to support newer csproj format with PackageReferences at all?

Also any plans to support packages.config at all?

**Comment by richorama** [comment]
Yes, I would like to support the newer formats. Thanks for raising the idea, I’ll put it on the todo list.
